### ISSUE TITLE
Specification include method

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#18]
```
I'd like to group tests as in scalax.testing:

===
object IntTests ...
object StringTests ...

object AllTests extends Specification {
  include(IntTests)
  include(StringTests)
}
===

it is easier syntax then

===
"all tests" isSpecifiedBy(
  IntTests, StringTests)
===
```

Original issue reported on code.google.com by `stepan.k...@gmail.com` on 7 May 2008 at 1:48

**Comment by GoogleCodeExporter** [comment]
```
Hi Stepan,

You can actually do:

===
object IntTests ...
object StringTests ...

object AllTests extends Specification {
  isSpecifiedBy(IntTests, StringTests)
}
===

And this should work.

Would that be ok for you?

Eric.

```

Original comment by `etorrebo...@gmail.com` on 7 May 2008 at 3:10
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This is better, but include method, that is added in r345 is better.
```

Original comment by `stepan.k...@gmail.com` on 7 May 2008 at 6:00

**Comment by GoogleCodeExporter** [comment]
```
Test added in r358. This will be available in release 1.2.7
```

Original comment by `etorrebo...@gmail.com` on 8 May 2008 at 9:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 13 May 2008 at 2:15
- Changed state: **Verified**
